azure-setup-guide/terraform/main.tf

```diff
@@ -0,0 +1,86 @@
+# Resource group setup
+resource "azurerm_resource_group" "xtdb_infra" {
```

Comment:
does this need the azurerm prefix? otherwise, save yourself a bit of verbosity :)

Reply:
Yeah, believe so - it's the [actual terraform name](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group) of the resource type we're creating, we're creating a ["Resource Group"](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/overview#resource-groups) in Azure as a result.